packages/cachy-browser-bin/cachy-browser-bin.pacscript

```diff
@@ -0,0 +1,34 @@
+name="cachy-browser-bin"
+gives="cachy-browser"
+pkgver="121.0-1"
+repology=("project: ${gives}")
+homepage='https://github.com/CachyOS/CachyOS-Browser-Settings'
+pkgdesc="Community-maintained fork of Firefox, focused on privacy, security and freedom"
+maintainer="vigress8 <vig@disroot.org>"
+arch=("amd64")
+gnuarch="x86_64"
+case $(
+  /lib64/ld-linux-x86-64.so.2 --help \
+    | grep 'x86-64.*supported' \
+    | head -1 \
+    | awk '{print $1}'
+) in
```

Comment:
I still don't like this. Is there no other way to check supported architectures?

Reply:
It's the way CachyOS recommends, I dunno. A quick search only finds that too

Reply:
I'm worried about how this will parse on the website end 😄

Reply:
ok so it returns `x86-64-v3` on the website end, so it's ok to merge, but in the future let's not do stuff like this 😆